import-service/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "deploy": "serverless deploy",
+    "test": "jest"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "aws-sdk": "^2.1185.0",
```

Comment:
Use this instead:
"@aws-sdk/client-s3": "^3.137.0",
"@aws-sdk/s3-request-presigner": "^3.137.0",

It will decrease the size of your lambdas to 4.1-5 vs 15mb

Reply:
thanks for the useful comments, applied all suggested changes